go/nbs/store.go

```diff
@@ -381,19 +380,12 @@ func (nbs *NomsBlockStore) Commit(current, last hash.Hash) bool {
 		return true
 	}
 
-	b := &backoff.Backoff{
```

Comment:
at least I think so...that definitely happens with batch gets. I thought the same happened with GetItem or whatever